pom.xml

```diff
@@ -23,18 +23,18 @@
 
   <properties>
     <java.version>17</java.version>
-    <mockwebserver.version>4.9.3</mockwebserver.version>
-    <okhttp3.version>4.9.1</okhttp3.version>
-    <postgresql.version>42.4.1</postgresql.version>
-    <spring-security.version>5.8.0</spring-security.version>
+    <mockwebserver.version>4.10.0</mockwebserver.version>
+    <okhttp3.version>4.10.0</okhttp3.version>
+    <postgresql.version>42.5.1</postgresql.version>
```

Comment:
This property isn't used and should be removed because we have only one Postgres dependency.

Reply:
You are right, I removed it